control/autoware_control_toolbox/CMakeLists.txt

```diff
@@ -0,0 +1,164 @@
+cmake_minimum_required(VERSION 3.10)
+project(autoware_control_toolbox)
+
+# FMT
+find_package(fmt)
+
+# require that dependencies from package.xml be available
+find_package(autoware_cmake REQUIRED)
+autoware_package()
+
+# -------------  FLAGS ---------------------------------------------------
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(
+    -Wall
+    -Wextra
+    -Wpedantic
+    # -Wuseless-cast
+    # -Wno-unused-parameter
+    # -Wno-unused-variable
+    # -Wno-narrowing
+    # -Wno-error=unused-but-set-variable
+    -Wno-conversion
+    -Wno-error=sign-conversion
+    -Wno-sign-compare
+    # -Wdouble-promotion
+    # -Wfloat-equal
+    -Waddress
+    -fvisibility=default
+    -fPIC) # visibility hidden or default is set here.
+endif()
```

Comment:
Custom flags should be removed.

Reply:
I put these flags because there are codes I can't change. For example, if I remove it, it gives an error for simple warnings. For example 

```
ent signedness: ‘Eigen::Index’ {aka ‘long int’} and ‘size_t’ {aka ‘long unsigned int’} [-Werror=sign-compare]
   64 |   for (Eigen::Index k = 0; k < npoints_; k++)
```
I am using Eigen::Index and std::vectors at the same time; for a simple loop, I need to introduce unnecessary static cast back and forth. 

If we can keep it, it would be appreciated.

Reply:
For this case, I fixed it. But the same sign comparison error will hit in the NMPC.